### ISSUE TITLE
ImportError: No module named pkgconfig

### ISSUE THREAD

**Issue by brad** [mehcode/python-xmlsec#39]
I can't install the latest version (1.0.1):
```
 % mkvirtualenv test
New python executable in /media/projects/.virtualenvs/test/bin/python
Installing setuptools, pip, wheel...done.
(test)  % which python
/media/projects/.virtualenvs/test/bin/python
(test)  % python -V
Python 2.7.12
(test)  % pip install xmlsec
Collecting xmlsec
  Downloading xmlsec-1.0.1.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-42z6jD/xmlsec/setup.py", line 5, in <module>
        import pkgconfig
    ImportError: No module named pkgconfig
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-42z6jD/xmlsec/
```

**Comment by marcus-ahle** [comment]
Running into this same issue as well when trying to pip install python3-saml. It worked when installing 0.6.1 of xmlsec, but now with 1.0.1 out it fails with the same `ImportError: No module named pkgconfig` error.

A direct pip install of 1.0.1 fails as well. 
python version: 3.4.4

**Comment by bgaifullin** [comment]
Hello, thank you for report. I will provide fix as soon as possible.
As workaround I can suggest this:
```
pip install pkgconfig lxml>=3.0
```

**Comment by bgaifullin** [comment]
fixed in version 1.0.4

**Comment by brad** [comment]
Wow! Thanks for the quick update @bgaifullin 👍
